wdl/PostprocessCCSedDemultiplexedSMRTCell.wdl

```diff
@@ -66,11 +68,11 @@ workflow PostprocessCCSedDemultiplexedSMRTCell {
 
     # associate barcode to various forms of sample ids
     Array[Pair[String, String]] barcode_2_folder = read_map(demuxed_barcode_2_folder)  # type coercion Map to Array[Pair] may only work in WDL 1.0
-    call Utils.SplitDelimitedString as get_barcodes {input: s = barcode_names, sep = ','}
```

Comment:
there is an internal wdl function called `sep` , so using `sep` as a variable confuses the wdl validator. So the variable name was changed here